src/App.test.tsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+  ReactDOM.unmountComponentAtNode(div);
```

Comment:
Could you please use react testing library for writing unit tests - https://testing-library.com/docs/react-testing-library/example-intro